### DISCUSSION TITLE
Object reference error

### DISCUSSION THREAD

**Discussion by sonic-linux** [marcinotorowski/MSIX-Hero#147] [Q&A]
I keep on seeing this error every time I open the app.
![Screenshot (1)](https://github.com/marcinotorowski/MSIX-Hero/assets/80959819/b48a798b-6404-458a-88e3-2f4a12b2112a)
I'm I doing something wrong or what's making it not work.
OS info:
Windows 11 23h2 build 22631.2792
Net 7.0 and 6.0 installed

**Comment by skycommand** [comment]
Experiencing the same.

Windows 10 v22H2
.NET 6.0 and 8.0 installed.

**Comment by marcinotorowski** [comment]
Should be fixed in 3.1 (https://github.com/marcinotorowski/MSIX-Hero/releases/tag/v3.1), please re-test

**Reply by skycommand** [reply]
Good to have you back 👍, and thanks for the job well done 🙏.
